lib/MusicBrainz/Server/WebService/Serializer/JSON/2/ReleaseGroup.pm

```diff
@@ -22,10 +22,14 @@ sub serialize
     $body{disambiguation} = $entity->comment // '';
 
     $body{'artist-credit'} = serialize_entity($entity->artist_credit, $inc, $stash)
-        if $entity->artist_credit && $inc && ($inc->artist_credits || $inc->artists);
```

Comment:
I hope removing the first condition doesn't cause `null` to be output in some cases, which would be a regression (similar to MBS-13518).

Reply:
In the test case for that ticket, we now output an artist credit rather than nothing at all (previously no AC was loaded). So it won't output `null` in that specific case, but I'm not 100% sure there aren't other cases, so I added the condition back.